hack/godep-save.sh

```diff
@@ -52,9 +52,23 @@ if [[ -d Godeps ]]; then
     mv Godeps "${BACKUP}/Godeps"
 fi
 
+## Workaround "httplex" was dropped from golang.org/x/net repo and the code
+## was moved to the "golang.org/x/net/http/httpguts" directory, we do not use
+## this directly, however many packages we vendor are still using the older
+## golang.org/x/net and we need to keep this until all those dependencies
+## are switched to newer golang.org/x/net.
+IGNORED_PACKAGES=(
+  "golang.org/x/net/lex/httplex"
+)
+REQUIRED_BINS=(
+  "golang.org/x/net/internal/nettest"
+  "golang.org/x/net/internal/socks"
+  "golang.org/x/net/internal/sockstest"
+)
+
 # Some things we want in godeps aren't code dependencies, so ./...
 # won't pick them up.
-REQUIRED_BINS=(
+REQUIRED_BINS+=(
```

Comment:
tiny suggestion: invert this logic so the workaround follows and appends, so we change less in the future to undo it? /shrug